### PR TITLE
Add final improvements to badges

### PR DESCRIPTION
**Pull request by L-Zuluaga**
Add small improvements to scorecards badge displayed name and changed the downloads badge for the general downloads one.

Also, included the templates for issues.

### CODE REVIEW

File: README.md
Comment (miguelnietoa) [inline README.md:1]: I think we could remove the line breaks at line 10 and 11

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+- [ ] Please disclose security vulnerabilities privately at [oss@kommit.co](mailto:oss@kommit.co).
+- [ ] Finally, be nice and have fun!.
+
+### Description
+
+A clear and concise description of what the bug is.
+
+### Environment
+
+* Operating system:
```
Comment (miguelnietoa) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md:22]: We could add Environment items specific to this project. Example: Ruby version

File: .github/workflows/ISSUE_TEMPLATE/._bug_report.md
Comment (miguelnietoa) [inline .github/workflows/ISSUE_TEMPLATE/._bug_report.md:1]: Please remove the files 
image

**Review by miguelnietoa — CHANGES_REQUESTED** [review]
@L-Zuluaga comments added

File: README.md
Comment (L-Zuluaga) [reply README.md:1]: I included that to create a small separation between the OpenSSF badges and the others. Do you think it's unnecessary?

Comment (miguelnietoa) [reply README.md:1]: Yes, IMO we could remove those line breaks and the OpenSSF badges will appear below

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+* Ruby version
```
Comment (miguelnietoa) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md:23]: Add `:`

**Review by miguelnietoa — CHANGES_REQUESTED** [review]

**Review by miguelnietoa — APPROVED** [review]
🚀

**Pull request merged**